controllers/snapshot/snapshot_adapter.go

```diff
@@ -477,8 +459,20 @@ func (a *Adapter) ensureEphemeralEnvironmentForScenarioExists(integrationTestSce
 	environment, ok := a.getExistingEnvironmentForScenario(integrationTestScenario, allEnvironments)
 	if !ok {
 		// environment doesn't exist, we have to create a new one
-		environment, err = a.createEnvironmentForScenario(integrationTestScenario)
+		existingEnv := findEnvironmentFromAllEnvironmentsByName(allEnvironments, integrationTestScenario.Spec.Environment.Name)
+		if existingEnv == nil {
+			err := h.NewEnvironmentNotInNamespaceError(integrationTestScenario.Spec.Environment.Name, integrationTestScenario.Namespace)
+			testStatuses.UpdateTestStatusIfChanged(
+				integrationTestScenario.Name, intgteststat.IntegrationTestStatusEnvironmentProvisionError,
+				fmt.Sprintf("Creation of copied ephemeral environment failed: %s. Try again after creating environment.", err))
```

Comment:
I think that we probably don't need this print here. If we do, it should be done using `a.logger.Info`.

Reply:
This is to add info to status annotation so user can see it.

Reply:
You are right, I misread this line, thanks.